src/main/resources/fabric.mod.json

```diff
@@ -26,8 +26,5 @@
   ],
   "depends": {
     "fabricloader": ">=0.7.4"
-  },
-  "suggests": {
-    "flamingo": "*"
```

Comment:
I can't find this mod in google.

Reply:
This was a sample mod that the example fabric mod has added, this shouldn't be present anymore due to it being pointless.